CHANGELOG.md

```diff
@@ -353,9 +353,15 @@ Released 2024-05-24
 - Extends `LambdaFWException` with extra property for easier handling edge case exceptions
 - Dumps `extra` data into the response when raising an error during handling requests
 
+### Version 0.6.6
+Released 2025-05-20
+
+- Updates python-jose to remove security vulnerability (PYSEC-2024-232,  PYSEC-2024-233)
```

Comment:
If the vulnerabilities are gone, can [we stop ignoring them](https://github.com/pdyba/lambdalizator/blob/fa1d26e88e4911f66f72dd5c21550587ec1936dd/Makefile#L102)? 🙂